debug-tools/amd-dbgapi.cmake

```diff
@@ -0,0 +1,515 @@
+# Copyright Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: MIT
+
+# Available defines to be used when building rocdbgapi:
+#
+# REQUIRE_THREADS
+#
+# Require linking against Threads. Default is OFF.
+#
+# If set to ON, include Threads when building. This is required when
+# rocdbgapi gets built within TheRock build platform due to a dependency
+# coming from amd_comgr.
+#
+# If building on its own, this does not need to be set.
+#
+#
+# API_TRACING
+#
+# Enable/Disable API tracing. Default is ON.
+#
+# For debugging and stepping through code, disabling API_TRACING is
+# recommended.
+#
+#
+# ENABLE_ASSERTIONS
+#
+# Enable/Disable assertions. Default is ON for DEBUG builds and OFF for
+# non-DEBUG builds.
+#
+#
+# BUILD_SHARED_LIBS
+#
+# Enable building ROCdbgapi either as a shared library os static library.
+# The default is ON, which builds ROCdbgapi as a shared library.
+#
+#
+# ROCdbgapi has the following dependencies:
+#
+# amd_comgr
+# libbacktrace (optional)
+# GIT (optional)
+# Doxygen (optional)
+# hsa-runtime64 headers
+# Threads (on request, see REQUIRE_THREADS)
+
+
+cmake_minimum_required(VERSION 3.8)
+
+project(amd-dbgapi VERSION 0.78.0)
```

Comment:
i guess this needs to be bumped manually each time the version in the submodule changes?